yarn.lock

```diff
@@ -1,5865 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file was totally removed. Is this change wanted?

Reply:
nope, it's a mistake, I'll fix it

Reply:
fixed in [`db88a8e` (#88)](https://github.com/pagopa/io-spid-commons/pull/88/commits/db88a8e958a2a08382a97d2c264928c93c868ba4)

Reply:
fixed in https://github.com/pagopa/io-spid-commons/pull/88/commits/cb562179b495421207f954da2d181d647857fee9